bin/yaml/cobol.yaml

```diff
@@ -0,0 +1,17 @@
+compilers:
+  cobol:
+    cobol:
+      type: tarballs
+      compression: tar
+      dir: gnucobol
+      url: https://sourceforge.net/projects/gnucobol/files/gnucobol/3.2/gnucobol-3.2-rc2.tar.bz2
```

Comment:
You may want to consider using the official download server https://alpha.gnu.org/gnu/gnucobol/ and _may_ also use another archive.

Reply:
I'm using `URL=https://sourceforge.net/projects/gnucobol/files/gnucobol/${MAJOR_MIN}/gnucobol-${VERSION}.tar.bz2` as the source currently. What do you mean "may also use another archive" ?

Reply:
I'll switch to URL=https://alpha.gnu.org/gnu/gnucobol/gnucobol-${VERSION}.tar.xz

Reply:
> I'll switch to URL=[https://alpha.gnu.org/gnu/gnucobol/gnucobol-${VERSION}.tar.xz](https://alpha.gnu.org/gnu/gnucobol/gnucobol-$%7BVERSION%7D.tar.xz)

... for PRE-Release-Versions, otherwise it would be https://ftp.gnu.org/gnu/gnucobol/ ;-)

>  What do you mean "may also use another archive" ?

some people prefer to not use .bz2 but .xz or .lz (from what I've gathered - the last one is the most superior one if you have an "up to date" system).

Reply:
Oh I see! Another archive format (not another website...I interpreted "archive" in this context as "archival website" :)